scripts/build.subr

```diff
@@ -537,6 +537,21 @@ buildworld () {
 # turn it back on afterwards.
 futurebuildworld () {
 	# Turn off ntpd if necessary
+	# Turn off salt too, if it's running, so it won't turn ntp back on
```

Comment:
This is a little confusing as we mention ntpd then salt, but the service invocations are in a different order, and nslcd isn't mentioned at all.